src/dialog/validator.js

```diff
@@ -0,0 +1,41 @@
+import _ from 'lodash'
+
+exports.validateFlowSchema = flow => {
+  const errorPrefix = `[Flow] Invalid flow "${flow && flow.location}"`
+
+  if (!flow || !_.isObjectLike(flow)) {
+    return 'Invalid JSON flow schema'
+  }
+
+  if (!flow.version || !_.isString(flow.version)) {
```

Comment:
Is it expected to me SemVer? If so we can use the `semver` module to validate and parse it.

Reply:
TBH not sure yet what will be the versioning strategy for the flows